validator_client/src/cli.rs

```diff
@@ -181,4 +181,18 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                     address of this server (e.g., http://localhost:5064).")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("disable-doppelganger-detection")
+                .long("disable-doppelganger-detection")
+                .value_name("DISABLE_DOPPELGANGER_DETECTION")
+                .help("By default, Lighthouse will delay startup for two epochs and monitor for \
+                    attestations on the network by any of the validators managed by this client. \
```

Comment:
```suggestion
                    messages on the network by any of the validators managed by this client. \
```

Only if we decided to check blocks and aggregates.